components/txn_types/src/lock.rs

```diff
@@ -266,7 +270,7 @@ impl Lock {
             size += 1 + size_of::<u64>() + MAX_VAR_U64_LEN;
         }
         if self.txn_source != 0 {
-            size += 2;
+            size += 1 + size_of::<u64>() + MAX_VAR_U64_LEN;
```

Comment:
Why is it sizeof u64 + MAX_VAR_U64_LEN ? Is there a mistake in `last_change_ts`? 😕 @sticnarf

Reply:
Ah, here it should be `1 + MAX_VAR_U64_LEN`

Reply:
The calculation in `last_change_ts` is because there are two integers (a fixed-length u64 and a var-length u64) after the flag.